### ISSUE TITLE
Middleware typing clashes with @types/request

### ISSUE THREAD

**Issue by jpike88** [tedeh/jayson#140]
```
const app = express();
const server = new jayson.Server(apiFunctions, { collect: false });
app.use(server.middleware());
```
Argument of type 'Function' is not assignable to parameter of type 'RequestHandlerParams'.

**Comment by jpike88** [comment]
Typescript tests should be extended to include middleware checks.

**Comment by tedeh** [comment]
Yes, this has been fixed in #4cf8475

**Comment by jpike88** [comment]
nice, when's next release?
